src/schema/v2/me/index.ts

```diff
@@ -574,20 +579,23 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "identityVerification",
       "unreadNotificationsCount",
       "unseenNotificationsCount",
-      "similarToRecentlyViewedConnection",
```

Comment:
Because we need to have fetched `me` in order to receive the `recently_viewed_artwork_ids` , we don't want to short-circuit for these fields.

Reply:
Cool makes sense!